src/utils/utils.js

```diff
@@ -37,19 +37,20 @@ export async function createMinimalApp(folderPath) {
 }
 
 export async function createTemplateApp(folderPath, template) {
-  const {
-    stdout
-  } = shelljs.exec(
-    `curl https://api.github.com/repos/sasjs/template_${template}`,
-    { silent: true }
+  const { stdout, stderr, code } = shelljs.exec(
+    `git ls-remote https://username:password@github.com/sasjs/template_${template}.git`,
```

Comment:
so the dummy username:password prevents the authentication prompt when the user provides the wrong template name?

Nice!